scripts/parse_specs.py

```diff
@@ -797,6 +802,31 @@ def _generate_ref(specs, tags, ref):
 
     return ref
 
+def refresh_enum_meta(obj, header, meta):
+    ## remove the existing meta records
+    if obj.get('class'):
+        meta['class'][obj['class']]['enum'].remove(obj['name'])
+        
+    del meta['enum'][obj['name']]
+    ## re-generate meta
+    meta = _generate_meta(obj, header['ordinal'], meta)
+
+
+def handle_enum_extensions(d, specs, header, meta):
+    matching_enum = [obj for s in specs for obj in s['objects'] if obj['type'] == 'enum' and d['name'] == obj['name']][0]
```

Comment:
Does this assume that the matching enum already exists? If so, it might be necessary to ensure that enum extensions are always processed last.